tests/containers/containerd_rmt.pm

```diff
@@ -0,0 +1,63 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Summary: Test deploy a helm chart in a k3s
+# - install k3s, kubectl and helm
+# - test helm repo add, update, search and show all
+# - add bitnami repo
+# - test helm install with apache helm chart
+# - test helm list
+# - check the correct deployment of the helm chart
+# - cleanup system (helm and k3s)
+#
+# Maintainer: qa-c team <qa-c@suse.de>
+
+use Mojo::Base 'publiccloud::basetest';
+use testapi;
+use utils;
+use version_utils qw(is_sle);
+use serial_terminal 'select_serial_terminal';
+use registration qw(add_suseconnect_product get_addon_fullname);
+use containers::k8s;
+
+sub run {
+    my ($self) = @_;
+
+    select_serial_terminal;
+    systemctl 'stop firewalld';
+    install_k3s();
+    install_kubectl();
+    install_helm();
+
+    my $rmt_helm = get_var('RMTTEST_REPO', 'https://gitlab.suse.de/brunoleon/rmt-helm/-/archive/main/rmt-helm-main.zip');
+    # pull in the testsuite
+    assert_script_run("wget --quiet --no-check-certificate " . $rmt_helm);
+    assert_script_run("unzip rmt-helm-main.zip");
+    my $myvalue = get_var('MY_VALUE', 'https://gitlab.suse.de/QA-APAC-I/testing/-/raw/master/data/rmtcontainer/myvalue.yaml');
+    assert_script_run("wget --no-check-certificate " . $myvalue);
+    record_info('chart', script_output('cat myvalue.yaml'));
```

Comment:
I am worried about the security of this. because in myvalue there is a password. In QE-C we managed the password in an other way. Ask us if you are interested

Reply:
Do you have the example, how to manage the password? Thanks.

Reply:
We have this function 
https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/22d08e8f7130030e80d073b31c5b7c7f7669e1d1/lib/publiccloud/utils.pm#L181 where we are getting the credentials from a server PUBLIC_CLOUD_CREDENTIALS_URL
e.g.
https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/22d08e8f7130030e80d073b31c5b7c7f7669e1d1/lib/publiccloud/aws_client.pm#L36

Reply:
Please contact me on the slack

Reply:
Removed myvalue file checking to not show password at openQA